app/views/shared/_navbar.html.erb

```diff
@@ -9,7 +9,7 @@
   <ul class="list-inline mx-auto justify-content-center">
     <li>
```

Comment:
@WhiteZh this commit has a different purpose than the PR itself, it would be best if you can please submit a separate PR for this Navbar fix.

Reply:
Understood. I have created a new pull request. Sorry, I am not very familiar with pull request system, it was a mistake.

Reply:
Also, I have moved this branch back to the dockerfile fixing commit. So the unifying heights commit is erased from this branch.

Reply:
Thank you so much